name.abuchen.portfolio/src/name/abuchen/portfolio/model/SecurityPrice.java

```diff
@@ -15,11 +16,12 @@
         @Override
         public int compare(SecurityPrice p1, SecurityPrice p2)
         {
-            return p1.date.compareTo(p2.date);
+            return p1.getDateTime().compareTo(p2.getDateTime());
         }
     }
 
     private LocalDate date;
+    private LocalDateTime dateTime;
```

Comment:
Added member variables to ```SecurityPrice``` is something I do not want to do.
Any change here has an impact on the XML serialization. And the ```SecurityPrice``` is the most used object in the XML.
Plus the time is not used in any of the calculations - PP does not do intra-day performance.

I have changed your implementation: the time is now held only internally in the ```HTMLTableQuoteFeed```. Before passing it to the outside, the feed keeps on the last price of a given day.